resources/views/dashboard/_widget.blade.php

```diff
@@ -12,7 +12,7 @@
         $background = Img::crop(600, 600)->url($widget->entity->image->path);
     }
 }
-if ($widget->conf('entity') && $widget->conf('entity') != 'all') {
+if ($widget->conf('entity') && $widget->conf('entity') != '') {
```

Comment:
Same here, !empty would be easier to read and support null values, not just empty strings (and be lighter on the database)